lib/src/generator/templates.aot_renderers_for_html.dart

```diff
@@ -1497,20 +1497,31 @@ String renderSidebarForContainer<T extends Documentable>(
       buffer.write(context1.href);
       buffer.write('''#instance-properties">Properties</a>
         </li>''');
-      var context6 = context1.publicInstanceFieldsSorted;
+      var context6 = context1.availableInstanceFieldsSorted;
       for (var context7 in context6) {
         buffer.writeln();
         buffer.write('''          <li''');
         if (context7.isInherited) {
           buffer.write(''' class="inherited"''');
         }
-        buffer.write('''>''');
+        buffer.write('''>
+            ''');
         buffer.write(context7.linkedName);
-        buffer.write('''</li>''');
+        if (context7.isProvidedByExtension) {
```

Comment:
This `if` statement appears to be duplicated several times in this file. I have to wonder whether we might be able to extract it into a utility method.

Reply:
SG. I've added a mustache partial for this text.

Reply:
Oops, Mustachio is apparently unable to handle this deduplication - an issue with referencing one partial multiple times from the same template, but each with different context types. I'm writing a unit test and I'll file a bug. For now I've left a TODO.